qa-pipelines/tasks/run-test.yml

```diff
@@ -8,5 +8,8 @@ inputs:
 - name: ci
 - name: s3.scf-config
 - name: pool.kube-hosts
+params:
```

Comment:
I'm wondering if it's a good idea to include these default params sections. If a pipeline is deployed without these values set from the config to template generation handoff, we want the pipeline deployment to fail, rather than having the concourse build fail after it gets run